### PR TITLE
New recipe: Template Numerical Toolkit

### PR DESCRIPTION
**Pull request by theclue**
TNT is a collection of templates and headers used to standardize numerical simulations written in C++. More informations are available [here](https://en.wikipedia.org/wiki/Template_Numerical_Toolkit). This is a very trivial recipe and I don't know if it fits HP policies.

This is intended as a starting point for providing Haiku with a set of basic, standard, tools for developers working on numerical processing. This toolkit is outdated on some parts, however, so using it or not today is open debate.

### CODE REVIEW

File: sci-libs/tnt/tnt-3_0_12.recipe [hunk 1]
```diff
@@ -0,0 +1,29 @@
+SUMMARY="Template Numerical Toolkit: C++ headers for array and matrices"
```
[the remark this answers is not in the document]
Comment (theclue) [reply sci-libs/tnt/tnt-3_0_12.recipe:1]: I know, but the source file have not. Is it ok to switch a dotted version number scheme but then to have the source zip file being hardcoded in the recipe, then?

[the remark this answers is not in the document]
Comment (extrowerk) [reply sci-libs/tnt/tnt-3_0_12.recipe:1]: Bash regex?

[the remark this answers is not in the document]
Comment (theclue) [reply sci-libs/tnt/tnt-3_0_12.recipe:1]: ehm....you're right i guess ^^'

[the remark this answers is not in the document]
Comment (extrowerk) [reply sci-libs/tnt/tnt-3_0_12.recipe:1]: Something like this would do the trick:
${portVersion//./_}
Untested and stolen from other recipe.

**Comment by Begasus** [comment]
@theclue could you remove the tnt-3_0_12.recipe  from this PR?

**Comment by theclue** [comment]
> @theclue could you remove the tnt-3_0_12.recipe from this PR?

file deleted; I did noticed it was still here in the remote repo. Sorry!!

**Comment by Begasus** [comment]
@theclue thanks! :)

**Pull request merged**